client/src/pages/write/Write.jsx

```diff
@@ -91,10 +98,10 @@ export default function Write() {
                                 />
                                 <input
                                     className="writeInput"
-                                    placeholder="Category"
+                                    placeholder="Enter catogories sparated by ','"
```

Comment:
We prefer avoiding using "Enter" in these placeholders as much as possible. The rest of the code seems to be fine.

Reply:
changed "Enter catogories sparated by ','" to "Catogories sparated by ','"